packages/react-app/src/app/app.tsx

```diff
@@ -26,6 +27,10 @@ const router = createBrowserRouter(
           element={<GiveawayDetailsPage />}
           loader={giveawayLoader}
         />
+        <Route
+          path="location/new"
```

Comment:
Just in case we had more routes to manage locations in the future
```suggestion
          path="locations/new"
```